nodejs/package.json

```diff
@@ -18,15 +18,15 @@
         "test": "mocha test/*.js",
         "buildProto": "pbjs -t static-module -w commonjs -o gtfs-realtime.js ../gtfs-realtime.proto"
     },
-    "engines": [
-        "node >= 0.10.0"
-    ],
+    "engines": {
+        "node": "18.12.1"
+    },
     "dependencies": {
-        "protobufjs": "6.8.6"
+        "protobufjs": "^6.11.3"
```

Comment:
Why not update to latest version (which is `7.1.2`) of protobufjs?

Reply:
@gauthier-th i had initially run into errors upgrading to `7.1.2`, which i was thinking of punting to a later PR. however, now i realize that the issue was just that `protobufjs` package no longer includes `pbjs`, and that i just needed to add `protobufjs-cli` package as an additional dependency to resolve the errors